src/components/TradeWidget/TxMessage.tsx

```diff
@@ -30,25 +31,49 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken })
 
   return (
     <TxMessageWrapper>
-      <p>
-        Your are selling {sellTokenAmount} {displaySellToken} for {displayReceiveToken}
-      </p>
-      <p>At a price of</p>
-      <p>
-        {priceInverse} {displaySellToken} per {displayReceiveToken}
-      </p>
-      <p>
-        {price} {displayReceiveToken} per {displaySellToken}
-      </p>
-      <p>
-        <strong>
-          You will receive at least {receiveTokenAmount} {displayReceiveToken}
-        </strong>
-      </p>
-      <p>
-        Your order starts {formatTimeInHours(validFrom || 0, 'now')}{' '}
-        {validUntil && `and will expire ${formatTimeInHours(validUntil, 'Never')}`}
-      </p>
+      <div className="intro-text">
+        <div>Carefully review the information below to make sure everything looks correct.</div>
+      </div>
+      <div className="message">
+        {/* Details */}
+        <div>
+          <strong>Order Details</strong>
+        </div>
+        <div>
+          Sell: <span>{sellTokenAmount}</span> <strong>{displaySellToken}</strong> for{' '}
+          <strong>{displayReceiveToken}</strong>
+        </div>
+        <div>
+          Receive: <UnderlinedText>at least</UnderlinedText> {receiveTokenAmount} <strong>{displayReceiveToken}</strong>
```

Comment:
That's not true if there's partial executions. 

How would you express better that:
```
You will receive at least XXX 567 GUSD if your order is completely executed
⚠️ Please, remember that your order can be also partially executed, or not executed at all. Read more about how Gnosis Protocol work [here].
```